index.md

```diff
@@ -39,4 +49,24 @@ redirect_from:
 - /engine/reference/commandline/cluster_rm/
 - /engine/reference/commandline/cluster_update/
 - /engine/reference/commandline/cluster_version/
+- /v1.4/
+- /v1.5/
+- /v1.6/
+- /v1.7/
+- /v1.8/
+- /v1.9/
+- /v1.10/
+- /v1.10/
```

Comment:
I'm not sure these would work; the current redirects match by prefix (so `/v1.4/engine/reference/commandline/cli/` (e.g.) would redirect to `/engine/reference/commandline/cli/` etc.

Reply:
Ohhh that's a very good point, didn't catch that we were using `KeyPrefixEquals`

Reply:
I guess we have to handle that with the lambda edge function I was working on.

Reply:
Yeah, we could add some of those I guess; then again, the current ones works, so perhaps no need to spend too much time on that for now